source/common/grpc/BUILD

```diff
@@ -72,6 +72,25 @@ envoy_cc_library(
 )
 
 envoy_cc_library(
+    name = "json_transcoder_filter_lib",
+    srcs = ["json_transcoder_filter.cc"],
+    hdrs = ["json_transcoder_filter.h"],
+    external_deps = [
+        "path_matcher",
+        "grpc_transcoding",
+    ],
+    deps = [
+        ":codec_lib",
+        ":common_lib",
+        ":transcoder_input_stream_lib",
+        "//include/envoy/http:filter_interface",
+        "//source/common/common:base64_lib",
+        "//source/common/http:headers_lib",
+        "@googleapis//:http_api_protos",
```

Comment:
We can't reference external deps like googleapis with "@googleapis", they need to appear in external_deps instead. We won't be able to perform the remapping during the Google import otherwise.